app/views/sprint-7/prototype-1/guidance/risk-1.html

```diff
@@ -65,13 +65,16 @@ <h2 class="govuk-heading-m" id="subsection-title">References</h2>
             <ul class="govuk-list govuk-!-font-size-16">
               <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```

Comment:
Something I missed but horizontal rules are not allowed as a child element of an unordered list. These need to be removed. Also the headings are not necessary, links are sufficient and would also follow the pattern of gov.uk https://www.gov.uk/guidance/information-requirements-for-pension-schemes-the-basics#relief-at-source-annual-return-of-information.

Reply:
Updated thanks @UsmanAfzal